includes/class-sensei-lesson.php

```diff
@@ -1671,10 +1671,7 @@ public function quiz_panel_question_feedback( $question_counter = 0, $question_i
 	}
 
 	public function question_get_answer_id() {
-		$data = $_POST['data'];
-		$answer_data = array();
-		parse_str( $data, $answer_data );
-		$answer = $answer_data['answer_value'];
+		$answer = $_GET['answer_value'];
```

Comment:
Here too.

Reply:
Added a fallback in https://github.com/Automattic/sensei/pull/2298/commits/4f82824ce78a62e3e57ec4fe301c32564ce95a77

From description:

> To test for backwards compatibility, copy the file `assets/js/lesson-metadata.min.js` from the `master` branch into this branch and test again. Everything should work, and you should see some "Doing it wrong" messages in the error log.